src/lib.rs

```diff
@@ -108,6 +108,9 @@ pub trait Core: Send + Sync {
     /// The returned activation is guaranteed to be for the same task queue / worker which was
     /// provided as the `task_queue` argument.
     ///
+    /// It is rarely a good idea to call poll concurrently. It handles polling the server
```

Comment:
This raises the question of when it is a good idea to poll concurrently.

Reply:
Good point. Probably never.... but never say never? I can change it to just say don't